README.md

```diff
@@ -1,3 +1,72 @@
 # Android Java Experiments
 
-This is general information about the project.
\ No newline at end of file
+This is general information about the project.
+
+# Android App For Learning Java
+
+## Table Of Contents
+* [About The Project](#about-the-project)
+* [Built With](#built-with)
+* [Installation And Setup](#installation-and-setup)
+    - [Virtual Android Phone Setup](#virtual-android-phone-setup)
+    - [Clone The Repository](#clone-the-repository)
+    - [Running The App In Virtual Phone](#running-the-app-in-virtual-phone)
+    - [Pulling And Committing Changes](#pulling-and-comitting-changes)
+
+## About The Project
+This project sets up Android Studio as an environment to learn Java code in an Android app.
+This will give anyone who wants to learn Java a more interactive way to do so.
+
+##### Experiments In This Project
+
+## Built With
+* Android Studio Flamingo | 2022.2.1
+
+## Installation and Setup
+Navigate to the [Android Studio](https://developer.android.com/studio?gclid=CjwKCAjw67ajBhAVEiwA2g_jEJTKBz_Uk-ZZz1ZZ67CIZlwE_kwawYZJcSdyhHUMscKrl4S6eIgrTBoC_F8QAvD_BwE&gclsrc=aw.ds) webpage and click the *Download Android Studio Flamingo* button. Accept the terms to the agreement that pops up and then click the download button. Once that installs, follow the steps detailed below to get the environment set up and ready to use.
```

Comment:
Just make the link https://developer.android.com/studio

Reply:
Fixed it.